rules/anti-static/obfuscation/js.yara

```diff
@@ -3,9 +3,14 @@ import "math"
 private rule probably_js {
   strings:
     $f_function = /function\(\w{0,8}\)/
-    $f_const    = "const" fullword
-    $f_return   = "return" fullword
-    $f_var      = "var" fullword
+    $f_const    = /\bconst\b/
+    $f_return   = /\breturn\b/
+    $f_var      = /\bvar\b/
+    $f_let      = /\blet\b/
+    $f_if       = /\bif\b/
```

Comment:
adding more 2-3 letter combinations is likely to raise our false-positive rate. Were there javascript files that you saw we weren't matching?

Reply:
Good point -- tweaked in [`ec3ea08` (#694)](https://github.com/chainguard-dev/malcontent/pull/694/commits/ec3ea08e526c90802b024fb0bb349cc4c070dc7a).